libr/core/cmd_info.c

```diff
@@ -481,18 +481,29 @@ static int cmd_info(void *data, const char *input) {
 			break;
 		case 'c': // for r2 `ic`
 			if (input[1] == '?') {
-				eprintf ("Usage: ic[ljq*] [class-index]\n");
+				eprintf ("Usage: ic[ljq*] [class-index or name]\n");
 			} else if (input[1] == ' ' || input[1] == 'q' || input[1] == 'j' || input[1] == 'l') {
 				RBinClass *cls;
 				RBinSymbol *sym;
 				RListIter *iter, *iter2;
 				RBinObject *obj = r_bin_cur_object (core->bin);
-				int idx = r_num_math (core->num, input + 2);
-				int count = 0;
 				if (obj) {
 					if (input[2]) {
+						int idx = -1;
+						const char * cls_name = NULL;
+						if (r_num_is_valid_input (core->num, input + 2)) {
+							idx = r_num_math (core->num, input + 2);
+						} else {
+							const char * first_char = input + ((input[1] == ' ') ? 1 : 2);
+							int not_space = strspn (first_char, " ");
```

Comment:
uhm, not sure how portable is `strspn`. its from C90, and seems to build fine with mingw, so let's keep it, but its the first time this is used in r2 iirc. :)